cc/toolchains/args/strip_flags/BUILD

```diff
@@ -0,0 +1,45 @@
+load("//cc/toolchains:args.bzl", "cc_args")
+load("//cc/toolchains:args_list.bzl", "cc_args_list")
+
+cc_args_list(
+    name = "strip_flags",
+    args = [
+        ":strip_debug",
+        ":preserve_dates",
+        ":output_file",
+        ":input_file",
+    ],
+    visibility = ["//visibility:public"],
+)
+
+cc_args(
+    name = "strip_debug",
```

Comment:
This appears to be unconditionally applied. I would have thought that this flag would be available, but not one we enable unconditionally? Maybe put it behind the feature flag debug or something like that?

Reply:
AFAIUI it's just that this is only called when bazel invokes the strip action, based on either `--strip=[always|sometimes|never]` or if you use the underlying `cc_binary.stripped` output target.

Reply:
Sorry, I mean that this is unconditionally applied when it takes the strip action. What I mean to say is that if I add `--strip=always` I probably wouldn't expect strip to always run with debug flags.

Reply:
I'm not sure I follow, this isn't particularly a debug flag but to strip all debug info, so with strip=always you'd definitely want this as the primary purpose.

But either way I just copied this from upstream:

https://github.com/bazelbuild/bazel/blob/90d4c0e348fe01794495dde8b74536a43afe124b/src/main/java/com/google/devtools/build/lib/rules/cpp/CppActionConfigs.java#L1360

AFAICT whenever strip is run it always passes these

Reply:
I misunderstood, thought it was a debug flag, sorry. LGTM then